src/stream/ngx_stream_proxy_module.c

```diff
@@ -742,6 +742,25 @@ ngx_stream_proxy_connect(ngx_stream_session_t *s)
 
     u->state->peer = u->peer.name;
 
+#if (NGX_STREAM_UPSTREAM_ZONE)
+    if (u->upstream && u->upstream->shm_zone
+        && (u->upstream->flags & NGX_STREAM_UPSTREAM_MODIFY)
+    ) {
```

Comment:
Style: in nginx oss there's no need to carry `)` to the next line.

Reply:
Thanks, fixed.